src/Altinn.App.Api/Controllers/DataController.cs

```diff
@@ -963,32 +1054,18 @@ await _instanceClient.UpdateDataValues(
 
     private ActionResult HandlePlatformHttpException(PlatformHttpException e, string defaultMessage)
     {
-        if (e.Response.StatusCode == HttpStatusCode.Forbidden)
-        {
-            return Forbid();
-        }
-        else if (e.Response.StatusCode == HttpStatusCode.NotFound)
-        {
-            return NotFound();
-        }
-        else if (e.Response.StatusCode == HttpStatusCode.Conflict)
+        return e.Response.StatusCode switch
         {
-            return Conflict();
-        }
-        else
-        {
-            return ExceptionResponse(e, defaultMessage);
-        }
+            HttpStatusCode.Forbidden => Forbid(),
+            HttpStatusCode.NotFound => NotFound(),
+            HttpStatusCode.Conflict => Conflict(),
+            _ => ExceptionResponse(e, defaultMessage)
+        };
     }
 
     private static bool InstanceIsActive(Instance i)
     {
-        if (i?.Status?.Archived != null || i?.Status?.SoftDeleted != null || i?.Status?.HardDeleted != null)
-        {
-            return false;
-        }
-
-        return true;
+        return i?.Status?.Archived is null && i?.Status?.SoftDeleted is null && i?.Status?.HardDeleted is null;
```

Comment:
With the `?` this reads a little funny. If `i` or `i.Status` are null then the instance is active?